### PR TITLE
Expand the Technical career to better capture the operational parts 

### PR DESCRIPTION
**Pull request by zupo**
Just a dump of ideas, needs polish

### CODE REVIEW

File: 5_People/career-technical.md [hunk 1]
```diff
@@ -6,40 +6,41 @@ The top two positions are two branches of the technical career:
...
-* Basic language features are mastered, but some advanced structures may still be unfamiliar.
+* Basic language/technology features are mastered, but some advanced approaches may still be unfamiliar.
 * Occasional issues following patterns and approaches within existing code bases.
 * Works mostly on clearly defined and scoped individual features or problems.
-* Usually less than 2 years of programming experience.
+* Usually less than 2 years of technical experience.
 
 
-## Junior Software Engineer
+## Junior Software Engineer / DevOps
```
Comment (nkk0) [inline 5_People/career-technical.md]: Could be `Junior DevOps`? Fine either way though.

File: 5_People/career-technical.md [hunk 2]
```diff
@@ -6,40 +6,41 @@ The top two positions are two branches of the technical career:
...
 
 * Work doesn't necessarily need to be reviewed, but general approach may be.
-* Follows established patterns and approaches within existing code bases with ease.
-* Is able to work on problems and features that are not clearly defined. Comes up with some specifications and API designs on their own.
-* Is able to take problem all the way from research, writing a User Story, implementation, and deployment.
-* Usually at least 2-5 years of programming experience.
+* Follows established patterns and approaches within established best practices bases with ease.
```
Comment (nkk0) [inline 5_People/career-technical.md]: This one sounds wrong now. Maybe the previous version but without `within`?
Like: `Follows established patterns and approaches existing code bases with ease.`
I don't think that one is too specific and I think it also applies to DevOps.

**Comment by nkk0** [comment]
I think it looks good!

**Comment by zupo** [comment]
@nkk0: I've read the whole document again and polished it. Pleas re-review.

I'd also like a review from @karantan and @dmurko. Others welcome as well.

File: 5_People/career-technical.md (hunk 1 above)
Comment (dmurko) [reply 5_People/career-technical.md]: @zupo I would agree here with Niko. The title can't be the same for two roles.

**Pull request merged**